e2etests/tests/validate_session.go

```diff
@@ -57,7 +57,7 @@ func validateSessionWithNeighbor(fromName, toName string, exec executor.Executor
 	}, 5*time.Minute, time.Second).ShouldNot(HaveOccurred())
 }
 
-func validateNoSuchNeigh(exec executor.Executor, neighborIP string) {
+func validateNoSessionForNeigh(exec executor.Executor, neighborIP string) {
```

Comment:
nit: validateNo**Established**SessionForNeigh ?

Reply:
can we change this? NoSession is a bit vague, where we want specifically no session in "Established"😅